.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
```

Comment:
In my opinion, we should enable `func-names`, perhaps with the "[as-needed](http://eslint.org/docs/rules/func-names#options)" parameter.  Naming a function is better for logs and debugging, which is super helpful in production -- or at least until we have a better logger (like #1519).

I think this should take care of all places that the name cannot be assumed.  To understand how Javascript figures out a function name, Dr. Axel Rauschmayer has a r[eally detailed article on it](http://2ality.com/2015/09/function-names-es6.html).